Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
+RUN openssl genrsa 2048 > ssl-key.pem
+RUN openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem
+
+COPY . /src
+RUN cd /src; npm install
+
+WORKDIR /src/data
+RUN echo $'@prefix n0: <http://www.w3.org/ns/auth/acl#>. \n\
```

Comment:
The `$'string \t with \n escape \t sequences'` syntax is bash-only and not particularly well known (so other developers coming along will stumble over it with a "what on earth does that do?" and waste a couple of minutes googling). Instead try using `echo -e 'string \t with \n escape \t sequences'`.

Additionally you should just commit the contents of this string as a file and `COPY` that file in. This will notably have trouble with the variable interpolation in the file (`$admin_user`), I'm looking into a way to work around that.